workers/data_refinery_workers/processors/smasher.py

```diff
@@ -189,6 +194,7 @@ def _smash(job_context: Dict) -> Dict:
         metadata['num_experiments'] = job_context["experiments"].count()
         metadata['aggregate_by'] = job_context["dataset"].aggregate_by
         metadata['scale_by'] = job_context["dataset"].scale_by
+        metadata['unsmashable_files'] = unsmashable_files
```

Comment:
The user will say this, so I think we should call this property "non_aggregated_files"